cip-0001.md

```diff
@@ -29,11 +29,11 @@ The CIP editors assign CIP numbers and change their status. Please send all CIP-
 
 The CIP process begins with a new idea for Counterparty. It is highly recommended that a single CIP contain a single key proposal or new idea. Small enhancements or patches often don't need a CIP and can be injected into the Counterparty development work flow with a patch submission to the Counterparty issue tracker. The more focused the CIP, the more successful it tends to be. The CIP editor reserves the right to reject CIP proposals if they appear too unfocused or too broad. If in doubt, split your CIP into several well-focused ones.
 
-Each CIP must have a champion -- someone who writes the CIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The CIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is CIP-able. Posting to the [Counterparty forums](https://counterpartytalk.org/c/development) or discussing it in the [Counterparty Slack](http://slack.counterparty.io/) is the best way to go about this.
+Each CIP must have a champion -- someone who writes the CIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The CIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is CIP-able. Posting to the [Counterparty Forums](https://forums.counterparty.io/c/development/), posting it in the [Counterparty Github Discussions](https://github.com/CounterpartyXCP/cips/discussions) or discussing it in the [Counterparty Developer Telegram](https://t.me/Counterparty_Dev) is the best way to go about this.
```

Comment:
Perhaps remove the link to the forums and just leave the link to the github discussions?